### PR TITLE
feat(snackbar): added snackbar to copy url

### PR DESCRIPTION
**Pull request by yishain11**

### CODE REVIEW

File: frontend/src/components/NewAgreement/AgreementCreatedSuccessfully.tsx [hunk 1]
```diff
@@ -14,26 +23,27 @@ const CopyIconWrapper = styled(SvgIcon)`
...
-      <CopyIconWrapper>
-        <CopyIcon />
-      </CopyIconWrapper>
-    </IconButton>
-  );
-};
-
 const AgreementCreatedSuccessfully: FC<{ agreementId: string }> = ({ agreementId }) => {
   const { t } = useTranslation();
+  const [snackbarOpen, setSnackbarOpen] = useState(false);
```
Comment (avirankatz) [inline frontend/src/components/NewAgreement/AgreementCreatedSuccessfully.tsx:28]: Suggestion: let's rename `snackbarOpen` to `isSnackbarOpen`

File: frontend/src/components/NewAgreement/AgreementCreatedSuccessfully.tsx [hunk 2]
```diff
@@ -81,6 +91,13 @@ const AgreementCreatedSuccessfully: FC<{ agreementId: string }> = ({ agreementId
             {AgreementText}
           </Typography>
           <CopyEL text={AgreementText} />
+          <Snackbar
+            open={snackbarOpen}
+            message="URL Copied Successfully"
```
Comment (avirankatz) [inline frontend/src/components/NewAgreement/AgreementCreatedSuccessfully.tsx:96]: Please use string bank

**Review by avirankatz — CHANGES_REQUESTED** [review]
I love the bite-sized PR! Please see the the requested changes

File: frontend/src/components/NewAgreement/AgreementCreatedSuccessfully.tsx (hunk 2 above)
Comment (yishain11) [reply frontend/src/components/NewAgreement/AgreementCreatedSuccessfully.tsx:96]: done

**Review by avirankatz — APPROVED** [review]

**Pull request merged**